packages/web-components/fast-foundation/src/slider/slider.template.ts

```diff
@@ -32,6 +32,9 @@ export const sliderTemplate: (
         <div part="positioning-region" class="positioning-region">
             <div ${ref("track")} part="track-container" class="track">
                 <slot name="track"></slot>
+                <div part="track-start" class="track-start" style="${x => x.position}">
+                    <slot name="track-start"></slot>
+                </div>
```

Comment:
This appears to be a breaking change - can we discuss the need/requirement for this?

Reply:
Is it breaking to add a new part? The requirement here was to add necessary elements to display the value, with the intent being to make it easy to customize the treatment if desired. For the latter reason I avoided using something like `before` or `after` elements to make it more formal. Especially since the `position` attribute used currently to set the edge makes some assumptions that I wanted to make more official in a future update. for instance, I'd like to add `track-end` as well, but was unable to style where it should "start" without modifying more of the code.

Reply:
@bheston with our update to support default slotted content, couldn't position be set on the default slotted content (which seems like it could be this)?

Reply:
It looks like the default slot is for `slider-labels`. Also, I had to bind the `position` in the template, which I'm not sure is possible through styling or without extending the `Slider` code. Even if it weren't used for labels, it seems that adding the position to the default slot would restrict its use.

Reply:
@bheston, sorry to be clear, the `track-start` slot could host the DIV in `fast-components` itself and bind to the data (it supports ViewTemplate's). An example of this is in Horizontal Scroll. I *think* that would keep this fairly non-breaking for anyone inheriting Foundation. By "default", I meant having content inside a slot by default.

Reply:
@bheston, I see that `track-start` is new - my bad. Let me look again

Reply:
@chrisdholt are we still holding on this PR?

Reply:
If we drop this template in Fluent and VSCode and neither breaks that's a good enough validation that this doesn't break implementations. If it renders incorrectly, yes we should hold for the visual refresh.

Reply:
Here's FAST before and after:

![image](https://user-images.githubusercontent.com/47367562/159096762-5b3627bf-33f1-49c6-bd6c-497a1092c60e.png)

Here's Fluent UI before and after:

![image](https://user-images.githubusercontent.com/47367562/159096782-a43b1ebc-9df6-4a34-8bba-b3eb4f8473d7.png)

It turns out the styles are already in place. Thanks copy and paste :)

VS Code didn't implement a slider, but now at least I'm setup ;)

Considering this is only additive I still see no reason it's not safe. We shouldn't be responsible for css selectors that assume too much about the underlying dom.